src/gui/chanOsc.cpp

```diff
@@ -146,6 +153,19 @@ void FurnaceGUI::drawChanOsc() {
           centerSettingReset=true;
         }
 
+        ImGui::TableNextColumn();
+        ImGui::Text("Automatic columns");
+        ImGui::SameLine();
+        ImGui::SetNextItemWidth(ImGui::GetContentRegionAvail().x);
+        const char* previewColType = autoColsTypes[chanOscAutoColsType];
+        if (ImGui::BeginCombo(" ", previewColType,0)) {
+          for (int j=0; j<4; j++) {
+            const bool isSelected=(chanOscAutoColsType==j);
+            if (ImGui::Selectable(autoColsTypes[j], isSelected)) chanOscAutoColsType=j;
```

Comment:
No comma between parameters in function call

Reply:
really?

```
[ 73%] Building CXX object CMakeFiles/furnace.dir/src/gui/chanOsc.cpp.o
~/furnacesrc/src/gui/chanOsc.cpp: In member function ‘void FurnaceGUI::drawChanOsc()’:
~/furnacesrc/src/gui/chanOsc.cpp:164:51: error: expected ‘)’ before ‘isSelected’
  164 |             if (ImGui::Selectable(autoColsTypes[j] isSelected)) chanOscAutoColsType=j;
      |                                  ~                ^~~~~~~~~~~
      |                                                   )
make[2]: *** [CMakeFiles/furnace.dir/build.make:3646: CMakeFiles/furnace.dir/src/gui/chanOsc.cpp.o] Error 1
make[1]: *** [CMakeFiles/Makefile2:194: CMakeFiles/furnace.dir/all] Error 2
make: *** [Makefile:156: all] Error 2
```

Reply:
Wait what? No! I mean no *space* between parameters!

Sorry!